packages/frontend/src/components/Strategies/Crab/CrabPositionV2.tsx

```diff
@@ -82,14 +123,56 @@ const CrabPosition: React.FC = () => {
         </Box>
       ) : (
         <Box display="flex" alignItems="center" gridGap="20px" marginTop="16px" flexWrap="wrap">
-          <Metric
-            label="Position value"
-            value={
-              <Typography className={clsx(classes.metricValue, classes.white)}>
-                {formatCurrency(currentCrabPositionValue.toNumber())}
-              </Typography>
-            }
-          />
+          {usdcQueued.isGreaterThan(0) ? (
+            <Metric
+              label={
+                <MetricLabel
+                  label="Initiated Deposit"
```

Comment:
Looks like there is a loading error here: 
- When I first got to crab page it showed me no position 
- After I did a standard deposit of 300 USDC, it showed me a position of $261 https://goerli.etherscan.io/tx/0x554cf53f50d08de70ae332e664e1181099feffa42fb9a5f933bff06ae4d46a43
<img width="1278" alt="Screen Shot 2022-12-06 at 11 05 55 PM" src="https://user-images.githubusercontent.com/13340486/206113696-748212ea-6a20-4ef2-b59c-32e64357747c.png">

- Was confused because initially thought it got my standard deposit wrong, but looks like there was an existing position from earlier 
- Went to withdraw to "instant" because I thought thats where I could dequeue since there was not "initaited deposit" with a cancel button, but that just flashWithdrew https://goerli.etherscan.io/tx/0xdd16feabb59084900670c0e08a9a08d6a2a47e5cda3883aac3f8b41f4473237c
- After the flashWithdraw it showed my queued deposit of 300 ETH with the "cancel button" but also my $261 position that should have been closed with the flashWithdraw